src/views/v2/commerce/checkout/purchaser-info/name.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Page Purchaser Info Name Field
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/purchaser-info/name.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this                  [Global] Template object.
+s */
+
+
+$label_classes = [
+	'tribe-common-b3',
+	'tribe-tickets__commerce-checkout-purchaser-info-name-field-label',
+];
+
+$field_classes = [
+	'tribe-tickets__commerce-checkout-purchaser-info-form-field',
+	'tribe-tickets__commerce-checkout-purchaser-info-form-field-name',
```

Comment:
This is missing: `tribe-common-form-control-text__input` and `tribe-tickets__form-field-input`.
Please refer to https://github.com/the-events-calendar/event-tickets-plus/blob/master/src/views/v2/components/meta/text.php to check the structure and generic classes.

Please [check the designs](https://www.figma.com/file/TvIXoUEJAGb0AptZI260UO/TicketsCommerce?node-id=1495%3A362) for better clarity.